_docs/_latest/setup/install.md

```diff
@@ -49,6 +49,7 @@ steps below:
 
       ```bash
       git clone https://github.com/GoogleCloudPlatform/forseti-security.git
+      cd forseti-security
```

Comment:
This step is focused on how to download the source code.  
So, the `cd` step should be better in the next next section, closer to the comment about `Make sure you are in the forseti-security folder`.

How about this?
```
       # Make sure you are in the forseti-security folder.
       # If the tag exists in the remote repository but you are unable to checkout the tag,
       # run command `git fetch --all` to fetch all the latest branch/tag information and run
       # the checkout command again.
       cd forseti-security
       git checkout tags/v2.0.0
```